source/core/parameters.cc

```diff
@@ -1302,7 +1302,7 @@ namespace Parameters
 
       prm.declare_entry(
         "expand particle-wall contact search",
-        "true",
+        "false",
         Patterns::Bool(),
         "Enables adding the boundary neighbor cells of boundary cells to the"
```

Comment:
Maybe add a more verbose comment for this parameter. (e.g. reproduce the text from teh warning).

Reply:
added